src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -8,71 +8,32 @@ namespace Microsoft.CodeAnalysis.Editor.Shared.Options
         /// future. Keeping the option while it's unimplemented allows all upgrade paths to
         /// maintain any customized value for this setting, even through versions that have not
         /// implemented this feature yet.
         /// </summary>
-        public static readonly PerLanguageOption2<bool> RenameTracking = new("FeatureOnOffOptions_RenameTracking", defaultValue: true);
+        public static readonly PerLanguageOption2<bool> RenameTracking = new("dotnet_feature_on_off_options_rename_tracking", defaultValue: true);
 
         /// <summary>
         /// This option is not currently used by Roslyn, but we might want to implement it in the
         /// future. Keeping the option while it's unimplemented allows all upgrade paths to
         /// maintain any customized value for this setting, even through versions that have not
         /// implemented this feature yet.
         /// </summary>
-        public static readonly PerLanguageOption2<bool> RefactoringVerification = new("FeatureOnOffOptions_RefactoringVerification", defaultValue: false);
-
-        public static readonly Option2<bool> NavigateAsynchronously = new("FeatureOnOffOptions_NavigateAsynchronously", defaultValue: true);
-
-        /// <summary>
-        /// This option was previously "bool?" to accomodate different supported defaults
-        /// that were being provided via remote settings. The feature has stabalized and moved
-        /// to on by default, so the storage location was changed to
-        /// TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2 (note the 2 suffix).
-        /// </summary>
-        public static readonly PerLanguageOption2<bool> AddImportsOnPaste = new("FeatureOnOffOptions_AddImportsOnPaste", defaultValue: true);
-
-        public static readonly Option2<bool?> OfferRemoveUnusedReferences = new("FeatureOnOffOptions_OfferRemoveUnusedReferences", defaultValue: true);
-
-        public static readonly Option2<bool> OfferRemoveUnusedReferencesFeatureFlag = new("FeatureOnOffOptions_OfferRemoveUnusedReferencesFeatureFlag", defaultValue: false);
-
-        public static readonly PerLanguageOption2<bool?> ShowInheritanceMargin = new("FeatureOnOffOptions_ShowInheritanceMargin", defaultValue: true);
-
-        public static readonly Option2<bool> InheritanceMarginCombinedWithIndicatorMargin = new("FeatureOnOffOptions_InheritanceMarginCombinedWithIndicatorMargin", defaultValue: false);
+        public static readonly PerLanguageOption2<bool> RefactoringVerification = new("dotnet_feature_feature_on_off_options_refactoring_verification", defaultValue: false);
 
-        public static readonly PerLanguageOption2<bool> InheritanceMarginIncludeGlobalImports = new("FeatureOnOffOptions_InheritanceMarginIncludeGlobalImports", defaultValue: true);
+        public static readonly Option2<bool> NavigateAsynchronously = new("dotnet_feature_on_off_options_navigate_asynchronously", defaultValue: true);
```

Comment:
This feature still have 'experiment' in the option page, so I am guessing it's still in experiment?